pycmds/hardware/filters.py

```diff
@@ -173,4 +232,4 @@ def set_motor_position(self, motor_position, units="deg"):
     Driver=Driver,
     GUI=GUI,
     Hardware=Hardware,
-)
+)
```

Comment:
newline at EOF?

Reply:
Technically, yes, black will put it, but it really does not matter that much...